assets/css/main.css

```diff
@@ -1,15 +1,37 @@
+@font-face {
+    font-family: gilroy-medium;
+    src: url("{{ site.baseurl }}/assets/font/Gilroy-Medium.otf") format("opentype");
```

Comment:
It looks perfectly fine on my machine (and not to mention great!), but I noticed that the file you added as part of this PR is a `.ttf` file and not `.otf` file, as specified here 🤷‍♀

Reply:
that's a good catch! it's updated